tools/refactoring/check-copyright.py

```diff
@@ -35,40 +22,11 @@ def print_error(mess, **kwargs):
     def print_warn(mess, **kwargs):
         print(mess, **kwargs)
 
-paths = sys.argv[1:]
-
-files = refactoring.find_files(paths, ['*'], excludes=excludespatterns)
-
-
-def replace(pattern, replacement):
-    matches = refactoring.find_matches(files, pattern)
-    print(matches.size())
-
-
-def createYearRegExp():
-    if (currentYear <= 2000):
-        print_error("Invalid current year: " + str(currentYear))
-        sys.exit(-1)
-    # regular expression should match any year up to current year
-    # the copy right year might consist of a single year, e.g. "2017"
-    # or a range of years "2010-2018"
-    #
-    # The resulting expression for currentYear = 2018 will have the form of:
-    #    r"^((?:20[0-1]\d-)?20[0-1][0-7]$)"
-    years = (currentYear - 1) % 10
-    decades = int((currentYear - 1) / 10) % 10
-    decadesStr = "[0-" + str(decades) + "]" if decades > 0 else "0"
-    yearsStr = "[0-" + str(years) + "]" if years > 0 else "0"
-
-    return re.compile(r"^((?:20" + decadesStr + r"\d-)?20" + decadesStr + yearsStr + r"$)")
-
 
 # match Inviwo Foundation copyright line
 copyrightLine_regex = re.compile(r"\s*[*#]\s+Copyright \(c\) (.*?) Inviwo Foundation\s*")
```

Comment:
So far only the comment indicators ` * ` and ` # ` are considered. Shouldn't `//` be part of that as well?
```suggestion
copyrightLine_regex = re.compile(r"\s*([*#]|//)\s+Copyright \(c\) (.*?) Inviwo Foundation\s*")
```

Reply:
We don't use that for the copyright block.